python/src/spark_rapids_ml/clustering.py

```diff
@@ -149,11 +149,10 @@ class KMeans(KMeansClass, _CumlEstimator, _KMeansCumlParams):
     tol: float (default = 1e-4)
         early stopping criterion if centers do not change much after an iteration.
 
-    featuresCol: str
-        the name of the column that contains input vectors. featuresCol should be set when input vectors are stored in a single column of a dataframe.
-
-    featuresCols: List[str]
```

Comment:
I see a Param with this name and a mixin (HasFeaturesCols) as well as getters and setters.   But it is not included as a constructor arg.   So are we getting rid of featuresCols throughout?   Or should we add as a constructor param.  @wbo4958 ?

Reply:
FWIW, I think this was originally trying to mirror the `HasInputCol` and `HasInputCols` [params](https://spark.apache.org/docs/latest/api/python/_modules/pyspark/ml/param/shared.html), but it is obviously redundant if `featuresCol` support both single str and list of str (which is a bit non-standard).  Anyhow, as long as we maintain compatibility w/ Pyspark code/usage, I think it's probably OK to remove.

Reply:
Seems we're using featuresCols to store the feature names, seems we can't remove it.